src/java_bytecode/java_class_loader.cpp

```diff
@@ -36,6 +36,12 @@ java_bytecode_parse_treet &java_class_loadert::operator()(
   queue.push("java.lang.Throwable");
   queue.push(class_name);
 
+  // Require often used library classes that implement widely used interfaces
```

Comment:
Change comment to note we're requiring *whatever the user asked for,* not a particular use-case